File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/SpecificationService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbSpecification;
import entity.PageResult;
import groupEntity.Specification;

public interface SpecificationService {

    //条件分页查询
    public PageResult search(Integer pageNum, Integer pageSize, TbSpecification tbSpecification);

    void add(Specification specification);

    Specification findOne(Long id);

    void update(Specification specification);
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/specificationService.js
app.service("specificationService", function ($http) {

    this.add = function (entity) {
        return $http.post("../specification/add.do", entity);
    }

    this.update = function (entity) {
        return $http.post("../specification/update.do", entity);
    }

    this.findOne = function (id) {
        return $http.get("../specification/findOne.do?id=" + id);
    }

    this.dele = function (ids) {
        return $http.get('../specification/delete.do?ids=' + ids);
    }
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post("../specification/search.do?pageNum=" + pageNum + "&pageSize=" + pageSize, searchEntity);
    }

    this.selectSpecList=function(){
        return $http.get("../specification/selectSpecList.do");
    }
});